CHANGELOG.rst

```diff
@@ -7,6 +7,14 @@ Change Log
 ----------
 
 
+6.6.0
+=====
+
+* In ``dcicutils.misc_utils``:
+
+  * Add ``keys_and_values_to_dict`` function (and associated unit test).
+
```

Comment:
I would not mind seeing a `dict_to_keys_and_values`, too.

Reply:
Good idea, all set.